checker/src/main/java/org/checkerframework/checker/optional/OptionalVisitor.java

```diff
@@ -491,6 +635,23 @@ public Void visitVariable(VariableTree tree, Void p) {
     return super.visitVariable(tree, p);
   }
 
+  /**
+   * Handles the case where a variable declaration contains "@NonEmpty".
```

Comment:
The text "handles" does not explain what this does.  Likewise for the method name.